bin/devenv

```diff
@@ -3,6 +3,10 @@
   set -x
 }
 
+if [ -z "${DEVENVROOT}" ] ; then
+  export DEVENVROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
```

Comment:
Forcifully set the root to the parent directory of the script being executed.

Reply:
Maybe you can try to source `scripts/init` before running devenv?
I remember  `DEVENVROOT` will be set in `init`

Reply:
> Maybe you can try to source `scripts/init` before running devenv?
> I remember `DEVENVROOT` will be set in `init`

If we need to place this command, we may want to update README to elaborate the usage : )

Reply:
Agree!  I think I should add some description about  devenv initialization into usage or README

Reply:
Ah it makes sense that ``scrpts/init`` should be sourced before running.

In this case, I will change the PR to check for the environment variable ``DEVENVROOT`` and exit if it's not set.